### PR TITLE
docs: fix missing sidebar when click on docmentation

### PR DESCRIPTION
**Pull request by qiweiii**
#545

The cause is that we removed 'rooks' page from sidebar, it corresponds to '/docs' route. Clicking on 'Documentation' brings use to '/docs', so i added it back to sidebar to fis it.

If we don't want rooks page at all. We need to change some other page to be '/docs'

### CODE REVIEW

**Comment by imbhargav5** [comment]
> The cause is that we removed 'rooks' page from sidebar, it corresponds to '/docs' route. Clicking on 'Documentation' brings use to '/docs', so i added it back to sidebar to fis it.

> If we don't want rooks page at all. We need to change some other page to be '/docs'
@qiweiii Yeah lets do that.

**Comment by qiweiii** [comment]
@imbhargav5 Which page do you want '/docs' routes to? I was thinking abt 'getting-started' page, but then both 'Getting started' and 'Documentation' will routes to 'getting'started'.

**Comment by imbhargav5** [comment]
@qiweiii Let's remove the unnecessary links and keep it simple. Good call!

File: sidebars.json [hunk 1]
```diff
@@ -3,10 +3,7 @@
     {
       "label": "Quick Start",
       "collapsed": false,
-      "items": [
-        "getting-started",
-        "motivation"
-      ],
+      "items": ["getting-started", "motivation", "rooks"],
```
Comment (imbhargav5) [inline sidebars.json:6]: I think rooks can be removed here. @qiweiii

**Review by imbhargav5 — CHANGES_REQUESTED** [review]

File: sidebars.json (hunk 1 above)
Comment (qiweiii) [reply sidebars.json:6]: If we remove it, the sidebar will disappear when to go 'docs/' page.

**Review by imbhargav5 — APPROVED** [review]

**Pull request merged**